test/asynchronous/test_cursor.py

```diff
@@ -1380,6 +1380,17 @@ async def test_getMore_does_not_send_readPreference(self):
         self.assertEqual("getMore", started[1].command_name)
         self.assertNotIn("$readPreference", started[1].command)
 
+    @async_client_context.require_sync
+    def test_to_list_when_empty(self):
+        self.db.drop_collection("test")
+
+        c = self.db.test.find().limit(0)
+        _ = c[0]
```

Comment:
What is the purpose of these two lines?

Reply:
Your comment suggested testing the case where the `self._empty == True`, which only happens when `__getitem__` is called in a case like this.

Reply:
I was referring to tailable cursors but sure the empty case should be tested too. In that case this test should be calling `c.to_list()` instead of calling to_list on a new cursor.